cross/python/PLIST

```diff
@@ -9,5 +9,4 @@ lnk:lib/libpython2.7.so
 lib:lib/libpython2.7.so.1.0
 rsc:lib/python2.7
 lib:lib/python2.7/lib-dynload/*.so
-rsc:etc/mime.types
```

Comment:
I wonder about this removal impact.
I guess (to confirm with git blame) that `mime.types` has been included here because needed by a module generated as wheel by an application (for instance)

Reply:
@Diaoul What do you think about this `etc/mime.types` removal in `cross/python` package ? Do you remember what which reason that file was required at the first place ? Thank you in advance for your help

Reply:
I think this file is required for the [`mimetypes` module](https://docs.python.org/2/library/mimetypes.html) to work.

Reply:
Aah, I will double check that and re-add if it indeed brakes this module.

Reply:
Confirmed, this patch is needed. Will re-add!